pkgs/applications/editors/emacs-modes/apheleia/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, emacs }:
+
+stdenv.mkDerivation {
+  name = "apheleia";
+
+  src = fetchFromGitHub {
+    owner = "raxod502";
+    repo = "apheleia";
+    rev = "3e342632b834a78f31ead48b94392e00dba1542c";
```

Comment:
Please track a tag.

Reply:
The latest rev of Apheleia has a fix for a fairly annoying bug (https://github.com/raxod502/apheleia/commit/f865c16), so I would prefer to use this instead.